src/index.html

```diff
@@ -8,10 +8,164 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+
+      <fieldset class="PI">
+        <legend>Personal information</legend>
+        <label class="form_field">
+            Surname:
+          <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+              required
+          >
```

Comment:
fix indents in all places, should be two spaces
```suggestion
        <label class="form_field">
          Surname:
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```